client/src/components/layout/MovieList.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react'
+
+const MovieList = (props) => {
+    const [movies, setMovies] = useState([])
+
+    const getMovies = async() => {
+        try {
+            const response = await fetch(`/api/v1/movies`)
+            if (!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const body = await response.json()
+            setMovies(body.movies)
+        } catch (error) {
+            console.error(`Error in fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getMovies()
+    },[])
+
+    const moviesListArray = movies.map((movie)=> {
+        return (
+            <>
+                <li key={movie.id}>
+                    {movie.title}
+                </li>
+                <br></br>
```

Comment:
for separation on the page, consider using CSS over `<br>` tags for consistency in styling